docs/overview/launch.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 4
+---
+
+# Launch
+
+Cyclic apps are hosted on AWS serverless infrastructure. This means any lambda constraints are inherited.
+
+- 30s Maximum request time
```

Comment:
Should organize this as limits somehow and maybe use the concept of launch to explain how there is no server to start or stop